src/pages/map.tsx

```diff
@@ -75,15 +75,17 @@ const Map: React.FC<Props> = (props) => {
   };
 
   const handleSchoolSearch = async (searchTerm: string) => {
-    const response = await fetch(
-      `/api/searchSchools?searchTerm=${searchTerm}`,
-    ).then((res) => res.json() as Promise<{ schools: School[] }>);
-
-    return response.schools.map((school) => ({
-      label: school.name,
-      value: school.name,
-      item: school,
-    }));
+    const results = props.schools
+      .filter(({ name }) =>
+        name.toUpperCase().includes(searchTerm.toUpperCase()),
+      )
+      .map((school) => ({
+        label: school.name,
+        value: school.name,
+        item: school,
+      }));
+
+    return results;
```

Comment:
maybe just return the .filter.map? no need for the results const.

Reply:
works for me!